src/main/java/org/openrewrite/kotlin/marker/KObject.java

```diff
@@ -26,10 +26,10 @@
 @With
 public class KObject implements Marker {
     UUID id;
-    Space prefix;
+    Space suffix;
 
-    public KObject(UUID id, Space prefix) {
+    public KObject(UUID id, Space suffix) {
```

Comment:
We have to be careful here. I am not 100% sure what this means for LST backward-compatibility. AFAIK Jackson should use the constructor, but I don't know if it would complain because the parameter name doesn't align with the serialized property name (`prefix` vs. `suffix`). I almost think you would have to test it.